.internal-ci/helm/fog-ledger/values.yaml

```diff
@@ -0,0 +1,234 @@
+      common:
+        tls:
+          clusterIssuer: letsencrypt-production-http
+        blocklist:
+          enabled: true
+          pattern: patterns/blocked-countries
+        annotations: |-
+          haproxy.org/server-ssl: "false"             # The backend (server) is http
+          haproxy.org/timeout-client: 239s            # 4 min timeout on azure
+          haproxy.org/timeout-server: 239s
+          haproxy.org/timeout-http-keep-alive: 120s
+          haproxy.org/abortonclose: "true"
+          haproxy.org/backend-config-snippet: |-
+            http-reuse aggressive
+            dynamic-cookie-key {{ include "fog-ledger.grpcCookieSalt" . }}
+            cookie "LEDGER" insert indirect nocache dynamic
+
+      grpc:
+        annotations: |-
+          haproxy.org/server-proto: "h2"              # Force GRPC/H2 mode
+
+      http:
+        annotations: |-
+          haproxy.org/path-rewrite: '/gw/(.*) /\1'    # Strip the /gw prefix
+
+    configMap:
+      data:
+        MC_OMAP_CAPACITY: '4194304'
+        MC_LEDGER_DB_URL: ''
+        MC_WATCHER_DB_URL: ''
+
+  store:
+    replicaCount: 1
+
+    image:
+      org: ''
+      name: fog-ledger
+      pullPolicy: Always
+
+    rust:
+      backtrace: full
+      log: info,rustls=warn,hyper=warn,tokio_reactor=warn,mio=warn,want=warn,reqwest=warn,rusoto_core=error,rusoto_signature=error,h2=error,rocket=warn,<unknown>=warn
+
+    persistence:
+      enabled: true
+      spec:
+        storageClassName: fast
+        accessModes:
+        - 'ReadWriteOnce'
+        resources:
+          requests:
+            storage: 512Gi
+
+    podAnnotations:
+      fluentbit.io/include: 'true' # collect logs with fluentbit
+      fluentbit.io/exclude-jaeger-agent: 'true'
+
+    # disable affinity rules for single node testing
+    podManagementPolicy: Parallel
+    affinityEnabled: true
+    topologySpreadConstraintsEnabled: true
+
+    ### Intel SGX extended resources are defined with: https://github.com/sebva/sgx-device-plugin
+    resources:
+      limits:
+        intel.com/sgx: 5000
+        memory: 5Gi
+      requests:
+        intel.com/sgx: 5000
+        memory: 5Gi
+        cpu: 1100m
+
+    nodeSelector:
+      sgx-enabled-node: 'true'
+
+    tolerations:
+    - key: sgx
+      operator: Equal
+      value: 'true'
+      effect: NoSchedule
+
+    configMap:
+      data:
+        MC_OMAP_CAPACITY: '4194304'
+        MC_LEDGER_DB_URL: ''
+        MC_WATCHER_DB_URL: ''
+
+  externalConfigMaps:
+    # sentry is optional
+    sentry:
+      name: sentry
+      ### required keys:
+      #   fog-ledger-sentry-dsn
+    mobilecoind:
+      name: fog-mobilecoind
+      ### required keys:
+      #   MC_PEER
+      #   MC_TX_SOURCE_URL
+      #   MC_QUORUM_SET
+
+  externalSecrets:
+    ias:
+      name: ias
+      ### required keys:
+      #   MC_IAS_API_KEY
+      #   MC_IAS_SPID
+
+    # ipinfo is optional
+    ipinfo:
+      name: ipinfo
+      ### required keys:
+      #   MC_IP_INFO_TOKEN
```

Comment:
Should these entries go away for now, they don't seem to be setting anything
```suggestion
```

Reply:
No, that's basically the documentation for the keys you need to populate when creating the external configMaps and secrets.